src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -266,8 +266,14 @@
 
 (defn fiat-formatted-for-ui
   [currency-symbol fiat-value]
-  (if (money/less-than fiat-value 0.01)
+  (cond
+    (money/equal-to fiat-value 0)
+    (str currency-symbol "0.00")
+
+    (money/less-than fiat-value 0.01)
     (str "<" currency-symbol "0.01")
+
+    :else
     (prettify-balance currency-symbol fiat-value)))
```

Comment:
Fixes issue if the balance is zero but it shows `< $0.01`

<img width="250" src="https://github.com/user-attachments/assets/846a7122-a051-4bcd-99fa-6e41d0a7dfa0" />